assets/airports/airportLoadList.js

```diff
@@ -257,6 +257,12 @@ window.AIRPORT_LOAD_LIST = (function() { // eslint-disable-line wrap-iife
             name: 'Vienna International Airport',
             wip: false
         },
+        {
+            icao: 'lrop',
+            level: 'medium',
+            name: 'Bucharest Henri Coanda International Airport',
+            wip: false
```

Comment:
I believe that `wip` should be set to `true` whilst it's being reviewed, or something like that @erikquinn can correct me if I'm wrong!